plugin/ui/completion.py

```diff
@@ -118,11 +91,15 @@ def current_completion(self) -> Optional[CopilotPayloadCompletion]:
         return self.completions[self.completion_index] if self.completions else None
 
     @property
-    def completion_style_type(self) -> Type[_BaseCompletion]:
-        return globals().get("_{}Completion".format(self.completion_style.capitalize()))
+    def completion_style_type(self) -> Type["_BaseCompletion"]:
```

Comment:
Strange, `pyright` complained when I tried `Type["_BaseCompletion"]`

Reply:
Maybe it's due to `globals().get(...)`, which too dynamic so pyright can't determinate whether it's `_BaseCompletion`?

Reply:
Probably